deploy/ray-operator/utils/util.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
+)
+
+// The function isCreated returns true if pod has been created and is maintained by the API server
+func IsCreated(pod *corev1.Pod) bool {
+	return pod.Status.Phase != ""
+}
+
+func Before(value string, a string) string {
```

Comment:
Header comment explaining what this does please

Reply:
Sure, I'll check the rest.